src/cmd/tools.go

```diff
@@ -109,4 +110,13 @@ func init() {
 	registryCmd.AddCommand(craneCmd.NewCmdPush(&cranePlatformOptions))
 	registryCmd.AddCommand(craneCmd.NewCmdCopy(&cranePlatformOptions))
 	registryCmd.AddCommand(craneCmd.NewCmdCatalog(&cranePlatformOptions))
+
+	syftCmd, err := cli.New()
+	if err != nil {
+		message.Fatal(err, "Unable to create syft CLI")
+	}
+	syftCmd.Use = "syft"
+	syftCmd.Short = "SBOM tools provided by Anchore Syft"
+	syftCmd.Aliases = []string{"s"}
```

Comment:
Do we want to include an alias called "sbom" (i.e. `zarf tools sbom ...`) as well so that it is more obvious what this is?

Reply:
Yes we do ❤️ feel free to push the change